tracing-subscriber/src/fmt/time/chrono_crate.rs

```diff
@@ -0,0 +1,177 @@
+use crate::fmt::format::Writer;
+use crate::fmt::time::FormatTime;
+
+use std::sync::Arc;
+
+/// Formats [local time]s and [UTC time]s with `FormatTime` implementations
+/// that use the [`chrono` crate].
+///
+/// [local time]: [`chrono::offset::Local`]
+/// [UTC time]: [`chrono::offset::Utc`]
+/// [`chrono` crate]: [`chrono`]
```

Comment:
I think this was supposed to be a module-level doc comment?
```suggestion
//! Formats [local time]s and [UTC time]s with `FormatTime` implementations
//! that use the [`chrono` crate].
//!
//! [local time]: [`chrono::offset::Local`]
//! [UTC time]: [`chrono::offset::Utc`]
//! [`chrono` crate]: [`chrono`]
use crate::fmt::format::Writer;
use crate::fmt::time::FormatTime;

use std::sync::Arc;
```

Reply:
...but, i guess having a module-level doc comment isn't super important since the module is private?